ember_debug/object-inspector.js

```diff
@@ -92,13 +91,16 @@ const emberNames = new Map([
   [NativeArray, 'NativeArray Mixin'],
   [Observable, 'Observable Mixin'],
   [ControllerMixin, 'Controller Mixin'],
-  [TargetActionSupport, 'TargetActionSupport Mixin'],
   [ActionHandler, 'ActionHandler Mixin'],
   [CoreObject, 'CoreObject'],
   [EmberObject, 'EmberObject'],
   [Component, 'Component'],
 ]);
 
+if (compareVersion(VERSION, '3.27.0') === -1) {
```

Comment:
It seems the deprecation was added in v3.27 according to the deprecation guides: https://deprecations.emberjs.com/v3.x#toc_ember-built-in-components-import